### ISSUE TITLE
[Bug]: Archived models need hiding

### ISSUE THREAD

**Issue by Terroosh** [BlafKing/sd-civitai-browser-plus#103]
### Describe the bug.

Archived models will show up in the model list but will produce an error if you click on them

Here's an example:
https://civitai.com/models/205503/tuttt

### Steps to reproduce the problem.

1. Add https://civitai.com/models/205503/tuttt to the browser
2. Click on the model
3. Error appears

### Expected behavior

Archived models should be hidden

### System info

* Extension version: 1.15.2
* OS: Win 11
* SD-WebUI version: 1.6
* Python: 3.10.6

### Console logs

```Shell
Traceback (most recent call last):
  File "K:\SD A1111\venv\lib\site-packages\gradio\routes.py", line 488, in run_predict
    output = await app.get_blocks().process_api(
  File "K:\SD A1111\venv\lib\site-packages\gradio\blocks.py", line 1431, in process_api
    result = await self.call_function(
  File "K:\SD A1111\venv\lib\site-packages\gradio\blocks.py", line 1103, in call_function
    prediction = await anyio.to_thread.run_sync(
  File "K:\SD A1111\venv\lib\site-packages\anyio\to_thread.py", line 33, in run_sync
    return await get_asynclib().run_sync_in_worker_thread(
  File "K:\SD A1111\venv\lib\site-packages\anyio\_backends\_asyncio.py", line 877, in run_sync_in_worker_thread
    return await future
  File "K:\SD A1111\venv\lib\site-packages\anyio\_backends\_asyncio.py", line 807, in run
    result = context.run(func, *args)
  File "K:\SD A1111\venv\lib\site-packages\gradio\utils.py", line 707, in wrapper
    response = f(*args, **kwargs)
  File "K:\SD A1111\extensions\sd-civitai-browser-plus\scripts\civitai_gui.py", line 692, in update_models_dropdown
    (html, tags, base_mdl, DwnButton, DelButton, filelist, filename, id, current_sha256, install_path, sub_folder) = _api.update_model_info(model_name,ret_versions['value'])
  File "K:\SD A1111\extensions\sd-civitai-browser-plus\scripts\civitai_api.py", line 694, in update_model_info
    model_url = model['downloadUrl']
KeyError: 'downloadUrl'
```

**Comment by BlafKing** [comment]
Thanks for the report!

This issue has now been fixed in v1.16! :)

**Comment by piwawa** [comment]
> Thanks for the report!
> 
> This issue has now been fixed in v1.16! :)

really? I am meeting this error:

![image](https://github.com/BlafKing/sd-civitai-browser-plus/assets/46926496/ba1883b2-179b-439f-a84a-d8d6157f8cac)

```
Feb 09 13:14:04 zernithos bash[2206762]:     return await future
Feb 09 13:14:04 zernithos bash[2206762]:   File "/home/bingo06/AIGC/stable-diffusion-webui/venv/lib/python3.10/site-packages/anyio/_backends/_asyncio.py", line 807, in run
Feb 09 13:14:04 zernithos bash[2206762]:     result = context.run(func, *args)
Feb 09 13:14:04 zernithos bash[2206762]:   File "/home/bingo06/AIGC/stable-diffusion-webui/venv/lib/python3.10/site-packages/gradio/utils.py", line 707, in wrapper
Feb 09 13:14:04 zernithos bash[2206762]:     response = f(*args, **kwargs)
Feb 09 13:14:04 zernithos bash[2206762]:   File "/home/bingo06/AIGC/stable-diffusion-webui/extensions/sd-civitai-browser-plus/scripts/civitai_file_manage.py", line 956, in file_scan
Feb 09 13:14:04 zernithos bash[2206762]:     preview_html = _api.update_model_info(None, model_versions.get('value'), True, id_value, api_response)
Feb 09 13:14:04 zernithos bash[2206762]:   File "/home/bingo06/AIGC/stable-diffusion-webui/extensions/sd-civitai-browser-plus/scripts/civitai_api.py", line 739, in update_model_info
Feb 09 13:14:04 zernithos bash[2206762]:     model_url = selected_version['downloadUrl']
Feb 09 13:14:04 zernithos bash[2206762]: KeyError: 'downloadUrl'
```

**Comment by piwawa** [comment]
> Thanks for the report!
> 
> This issue has now been fixed in v1.16! :)

version: [v1.7.0](https://github.com/AUTOMATIC1111/stable-diffusion-webui/commit/cf2772fab0af5573da775e7437e6acdca424f26e)  •  python: 3.10.6  •  torch: 2.2.0+cu118  •  xformers: 0.0.24+cu118  •  gradio: 3.41.2  •  checkpoint: [e3edb8a26f](https://google.com/search?q=e3edb8a26f44dfddc124129004b5ded3378be974b8d3bf82f43b9660d416d3d1)

![image](https://github.com/BlafKing/sd-civitai-browser-plus/assets/46926496/7ea1e132-05b7-443d-84bb-e4110134cb28)

**Comment by BlafKing** [comment]
Thanks for posting @piwawa, this bug was re introduced since the `update_model_info` function now gets used during model updating, which it did not do before v3.3.0

I'll try to include a hotfix for this issue in the upcoming update but it will be a little bit of guess work since I do not know which model caused this error, would help if you could post which model you encountered this error on! :)
